rotkehlchen/tests/unit/decoders/test_gnosis_sdai.py

```diff
@@ -0,0 +1,244 @@
+            location_label=user_address,
+            notes=f'Withdraw {received_amount} XDAI from sDAI contract',
+            tx_hash=tx_hash,
+            address=receive_address,
+            counterparty=CPT_SDAI,
+        ), EvmEvent(
+            sequence_index=1029,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=Asset('eip155:100/erc20:0xaf204776c7245bF4147c2612BF6e5972Ee483701'),
+            balance=Balance(amount=FVal(sent_amount)),
+            location_label=user_address,
+            notes=f'Return {sent_amount} sDAI to sDAI contract',
+            counterparty=CPT_SDAI,
+            tx_hash=tx_hash,
+            address=ZERO_ADDRESS,
+        ),
+    ]
+
+    assert actual_events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('gnosis_accounts', [['0x5938852FE18Ad6963322FB98D1fDDA5c24DD8a0E']])
+def test_deposit_wxdai_to_sdai(database, gnosis_inquirer, gnosis_accounts):
+    user_address = gnosis_accounts[0]
+    tx_hash = deserialize_evm_tx_hash('0xd406f40ecd2538d41adb2e645c8fb6d32cec5485510798bfed5d991c258d4b1d')  # noqa: E501
+    actual_events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=gnosis_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1706794335000)
+    withdraw_amount, deposit_amount, gas_amount = '319.006747127200240848', '331.313258668881367296', '0.0006162151'  # noqa: E501
+    expected_events = [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_XDAI,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burned {gas_amount} XDAI for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1267,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_WXDAI,
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=user_address,
+            notes=f'Deposit {deposit_amount} WXDAI into the sDAI contract',
+            tx_hash=tx_hash,
+            counterparty=CPT_SDAI,
+            address='0xD499b51fcFc66bd31248ef4b28d656d67E591A94',
+        ), EvmEvent(
+            sequence_index=1269,
+            timestamp=timestamp,
+            location=Location.GNOSIS,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=Asset('eip155:100/erc20:0xaf204776c7245bF4147c2612BF6e5972Ee483701'),
```

Comment:
You can create a global variable in this file that is 
```
A_GNOSIS_SDAI = evm_address_to_identifier(
            address=GNOSIS_SDAI_ADDRESS,
            chain_id=ChainID.GNOSIS,
        )
```

and use it in all the tests that you use it